starters/next13-zustand-bulma/.storybook/with-font-decorator.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { Open_Sans } from 'next/font/google';
+
+const opensans = Open_Sans({ subsets: ['latin'], weight: 'variable' });
+
+export const withFontDecorator = (Story, context) => {
+  document.body.classList.add(opensans.className);
+  return <Story {...context} />;
+};
```

Comment:
A note for later:
I think we should add a sentence or two about this google font and decorator stuff because I believe this is new for Next 13